src/Acuminator/Acuminator.Analyzers/StaticAnalysis/DacPropertyAttributes/MultipleAttributesOnDacPropertyFixBase.cs

```diff
@@ -0,0 +1,171 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Composition;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Threading;
+using System.Threading.Tasks;
+using Acuminator.Utilities.Common;
+using Acuminator.Utilities.Roslyn;
+using Acuminator.Utilities.Roslyn.PXFieldAttributes;
+using Acuminator.Utilities.Roslyn.Syntax;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CodeActions;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Text;
+
+namespace Acuminator.Analyzers.StaticAnalysis.DacPropertyAttributes
+{
+	/// <summary>
+	/// A multiple attributes on DAC property fix base class.
+	/// </summary>
+	public abstract class MultipleAttributesOnDacPropertyFixBase : CodeFixProvider
+	{
+		public override FixAllProvider GetFixAllProvider() => WellKnownFixAllProviders.BatchFixer;
+
+		public override async Task RegisterCodeFixesAsync(CodeFixContext context)
+		{
+			string codeActionName = GetCodeActionName();
+			Func<FieldTypeAttributeInfo, bool> removePredicate = GetRemoveAttributeByAttributeInfoPredicate();
+
+			if (codeActionName.IsNullOrWhiteSpace() || removePredicate == null)
+				return;
+
+			SyntaxNode root = await context.Document.GetSyntaxRootAsync(context.CancellationToken)
+													.ConfigureAwait(false);
+
+			if (!(root?.FindNode(context.Span) is AttributeSyntax attributeNode) || context.CancellationToken.IsCancellationRequested)
```

Comment:
`ThrowIfCancellationRequested`?

Reply:
@dropsonic this is a code executed before registration of the code fix. Should we throw here? Will it be handled correctly?

Reply:
ok, I'll fix it